Refactor of Tagging Section (#7516)
DataDog/documentation: content/en/getting_started/integrations/_index.md

      path: /var/log/apache2/error.log
      source: apache
      sourcecategory: http_web_access

>       service: apache

Reviewer: this should not be here under logs but under the instance section. Otherwise its not unified

Reviewer 2: yup, or `init_config`

Reviewer 3: ➕ 
Just to clarify, to have it really unified in our docs, we should define it under `init_config`, i.e.:

```yaml
init_config:
  service: apache
```

and remove all occurences of `service: apache` from the `logs` section.

Ideally, we could add a comment about `service` being supported under `init_config` only since Agent v7.18/v6.18

### Tagging

Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. For more information about tagging, see [Getting started with tags][25].

If you define tags in the `datadog.yaml` file, the tags are applied to all of your integrations data. Once you've defined a tag in `datadog.yaml`, all new integrations inherit it.

For example, setting `service` in your config file is the recommended [agent setup][26] for monitoring separate, independent systems.

> However, to unify your environment, it is recommended to configure `service` based on your Agent version. To learn how more about unified service tagging, refer to the dedicated [unified service tagging][12] documentation.

Author: ```suggestion
To better unify your environment, it is also recommended to configure the `env` tag in the Agent. To learn more about unified service tagging, refer to the dedicated [unified service tagging][12] documentation.
```

### Tagging

> Tagging is a key part of filtering and aggregating the data coming into Datadog across many sources. For more information about tagging, see [Getting started with tags][25].

Reviewer: I cannot find information about the `service` config option in integrations creating the service tag. This is the preferred way for traditional bare metal agent setups that are monitoring separate, independant, systems.

Reviewer 4: Good point, we should add a more specific call out for setting `service`.

`env` would likely be configured via the agent. `version` _could_ be set but then folks would need to update their config + restart the agent every time they change the tag.

Reviewer 2: Here are the PRs btw:

- https://github.com/DataDog/datadog-agent/pull/4877
- https://github.com/DataDog/datadog-agent/pull/4895
- https://github.com/DataDog/jmxfetch/pull/269
